scss/components/_inline-datepicker.scss

```diff
@@ -0,0 +1,49 @@
+$form-control-box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075);
+$inline-error-bg: #fbe3e4;
```

Comment:
Cant we use existing colors? Also check if the values are already present in either civicase or shoreditch.
Ideally we should not be introducing new colors, because the UI elements we are making already exist(like generic datepicker or input fields).

If we cant at all, lets atleast move all variables to the `_variables.scss` file.